commands/src/main/java/org/wildfly/extras/creaper/commands/logging/Level.java

```diff
@@ -0,0 +1,35 @@
+package org.wildfly.extras.creaper.commands.logging;
+
+public enum Level {
+
+    ALL("ALL"),
+    FINEST("FINEST"),
+    FINER("FINER"),
+    TRACE("TRACE"),
+    DEBUG("DEBUG"),
+    FINE("FINE"),
+    CONFIG("CONFIG"),
+    INFO("INFO"),
+    WARN("WARN"),
+    WARNING("WARNING"),
+    ERROR("ERROR"),
+    SEVERE("SEVERE"),
+    FATAL("FATAL"),
+    OFF("OFF");
```

Comment:
`DEBUG` is same as `FINER` and `FINE` according to http://docs.jboss.org/mobicents/sip-presence/1.0.0.FINAL/html/logging_traces_and_alarms.html#tracers_vs_log4j